### PR TITLE
Squash dat getter-setter non-sense.

### PR DESCRIPTION
**Pull request by al3x-edge**
Removes deprecation for using the same function as getter and setter.

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: addon/services/intl.js [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import Ember from 'ember';
 import { IntlMessageFormat } from '../utils/data';
 import createFormatCache from '../format-cache/memoizer';
+import computedPolyfill from 'ember-new-computed';
```
Comment (jasonmit) [inline addon/services/intl.js]: call it `computed` and replace all instances of Ember.computed with ember-new-computed :+1:

**Comment by al3x-edge** [comment]
@jasonmit Updated to change all references to `Ember.computed` to `computed` from `ember-new-computed`.

**Comment by jasonmit** [comment]
Thanks again Alex :) :+1:

**Comment by al3x-edge** [comment]
Not a problem :+1:

**Pull request merged**